packages/discord.js/src/structures/GuildScheduledEvent.js

```diff
@@ -192,12 +192,13 @@ class GuildScheduledEvent extends Base {
   }
 
   /**
-   * The time the guild scheduled event will start at
-   * @type {Date}
+   * The time the guild scheduled event will start at,
+   * or `null` if the event does not have a scheduled time to start
```

Comment:
> if the event does not have a scheduled time to start

This leads one to believe that events may not require a scheduled time to start. The info block on `GuildScheduledEvent#scheduledStartTimestamp` seems to be a more accurate description of the nullable behaviour - can that be put here too?

Reply:
I wasn't sure if I should duplicate the notice, I'll change it.